.github/workflows/shellcheck.yml

```diff
@@ -31,4 +31,11 @@ jobs:
         with:
           check_together: 'yes'
           ignore: test_scripts 
-        
+  BatsTests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run Bats
+        run: |
+          cd $GITHUB_WORKSPACE
+          docker build -f ./test_scripts/unit_tests/Dockerfile -t bats_tests . --no-cache --progress=plain && docker run bats_tests
```

Comment:
It's probably not important but I'd normally do a `docker run --rm ...` just so it auto-removes the container once done.

Reply:
Ahhh... you're right, I actually was doing --rm locally